dbt/seeds/etl_full_row_counts.csv

```diff
@@ -1177,7 +1177,7 @@ core_eia923__monthly_fuel_receipts_costs,2021,37183
 core_eia923__monthly_fuel_receipts_costs,2022,38792
 core_eia923__monthly_fuel_receipts_costs,2023,36835
 core_eia923__monthly_fuel_receipts_costs,2024,36356
-core_eia923__monthly_fuel_receipts_costs,2025,13263
+core_eia923__monthly_fuel_receipts_costs,2025,15288
```

Comment:
is it weird that the Q3 count is less than half the annual count from past years? I don't know what the normal reporting lag is -- if this number will catch up over the next few quarters then nw